ERROR - 2024-12-09 06:13:20 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 06:13:20 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 06:14:16 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 06:14:16 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 06:14:18 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 06:14:18 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 06:15:22 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 06:15:22 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 06:16:23 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 06:16:23 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 06:16:25 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 06:16:25 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 06:16:25 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 06:16:25 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 06:24:20 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(325): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 06:24:20 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name` AS `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(325): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-12-09 06:46:09 --> Class "App\Controllers\AddPurchaseModel" not found
in APPPATH\Controllers\Asset.php on line 384.
 1 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purchase_reasons()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-12-09 06:46:24 --> Class "App\Controllers\AddPurchaseModel" not found
in APPPATH\Controllers\Asset.php on line 384.
 1 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purchase_reasons()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:08:50 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:08:50 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:11:18 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:11:18 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:11:19 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:11:19 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:11:19 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:11:19 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(327): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:21:01 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:21:01 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:21:02 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:21:02 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:29:45 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:29:45 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:29:46 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:29:46 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:31:33 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:31:33 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:31:38 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:31:38 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:31:39 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:31:39 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:33:14 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:33:14 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 07:35:53 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 07:35:53 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 08:36:36 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 08:36:36 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 08:39:33 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 08:39:33 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 08:41:20 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 08:41:20 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 08:41:21 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 08:41:21 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 08:53:15 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 08:53:15 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 08:54:05 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 08:54:05 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 08:55:30 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 08:55:30 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 09:02:30 --> mysqli_sql_exception: Unknown column 'purpose_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `purpose...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `purpose...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `purpose...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 09:02:30 --> Unknown column 'purpose_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `purpose_name`
FROM `newasset`
WHERE `deleted_on` IS NULL
GROUP BY `purpose_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(145): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_purposes()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 10:00:37 --> mysqli_sql_exception: Unknown column 'uom_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `uom_nam...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `uom_nam...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `uom_nam...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_nam...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(228): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 10:00:37 --> Unknown column 'uom_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_name`
FROM `uom`
GROUP BY `uom_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(228): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 10:00:38 --> mysqli_sql_exception: Unknown column 'uom_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `uom_nam...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `uom_nam...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `uom_nam...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_nam...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(228): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 10:00:38 --> Unknown column 'uom_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_name`
FROM `uom`
GROUP BY `uom_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(228): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 10:00:59 --> mysqli_sql_exception: Unknown column 'uom_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `uom_nam...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `uom_nam...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `uom_nam...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_nam...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(228): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 10:00:59 --> Unknown column 'uom_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_name`
FROM `uom`
GROUP BY `uom_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(228): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 10:08:31 --> mysqli_sql_exception: Unknown column 'uom_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `uom_nam...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `uom_nam...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `uom_nam...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_nam...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(235): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 10:08:31 --> Unknown column 'uom_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_name`
FROM `newasset`
WHERE `uom_name` IS NOT NULL
AND `deleted_on` IS NULL
GROUP BY `uom_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(235): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 10:08:32 --> mysqli_sql_exception: Unknown column 'uom_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `uom_nam...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `uom_nam...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `uom_nam...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_nam...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(235): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 10:08:32 --> Unknown column 'uom_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_name`
FROM `newasset`
WHERE `uom_name` IS NOT NULL
AND `deleted_on` IS NULL
GROUP BY `uom_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(235): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-12-09 10:09:14 --> mysqli_sql_exception: Unknown column 'uom_name' in 'field list' in C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\viyoma\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `uom_nam...', 0)
#1 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `uom_nam...')
#2 C:\xampp\htdocs\viyoma\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `uom_nam...')
#3 C:\xampp\htdocs\viyoma\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_nam...', Array, false)
#4 C:\xampp\htdocs\viyoma\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\viyoma\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\viyoma\app\Controllers\Asset.php(235): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
#8 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
#9 C:\xampp\htdocs\viyoma\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\viyoma\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-12-09 10:09:14 --> Unknown column 'uom_name' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `uom_name`
FROM `newasset`
WHERE `uom_name` IS NOT NULL
AND `deleted_on` IS NULL
GROUP BY `uom_name`', [], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Asset.php(235): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->all_uoms()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-12-09 10:33:25 --> Undefined variable $dealer
in APPPATH\Views\uom\view.php on line 1869.
 1 APPPATH\Views\uom\view.php(1869): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $dealer', 'C:\\xampp\\htdocs\\viyoma\\app\\Views\\uom\\view.php', 1869)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\viyoma\\app\\Views\\uom\\view.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('uom/view', [], true)
 5 APPPATH\Controllers\Uom.php(56): view('uom/view', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Uom->view()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Uom))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-12-09 10:52:27 --> Undefined property: App\Controllers\Asset::$AddPurchaseModel
in APPPATH\Controllers\Asset.php on line 36.
 1 APPPATH\Controllers\Asset.php(36): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Asset::$AddPurchaseModel', 'C:\\xampp\\htdocs\\viyoma\\app\\Controllers\\Asset.php', 36)
 2 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->index()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-12-09 10:52:48 --> Undefined property: App\Controllers\Asset::$AssetModel
in APPPATH\Controllers\Asset.php on line 36.
 1 APPPATH\Controllers\Asset.php(36): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Asset::$AssetModel', 'C:\\xampp\\htdocs\\viyoma\\app\\Controllers\\Asset.php', 36)
 2 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->index()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-12-09 11:17:41 --> Undefined variable $base
in APPPATH\Views\newasset\edit.php on line 2201.
 1 APPPATH\Views\newasset\edit.php(2201): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $base', 'C:\\xampp\\htdocs\\viyoma\\app\\Views\\newasset\\edit.php', 2201)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\viyoma\\app\\Views\\newasset\\edit.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('newasset/edit', [], true)
 5 APPPATH\Controllers\Asset.php(358): view('newasset/edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Asset->edit('VGtSQlBRPT0=')
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Asset))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
